Cargo.toml

```diff
@@ -41,15 +42,20 @@ arrow-data = "51.0.0"
 arrow-ipc = "51.0.0"
 arrow-schema = "51.0.0"
 arrow-select = "51.0.0"
+async-trait = "0.1"
 bindgen = "0.69.4"
 bytes = "1.6.0"
 bzip2 = "0.4.4"
 cargo_metadata = "0.18.1"
 criterion = { version = "0.5.1", features = ["html_reports"] }
 croaring = "1.0.1"
 csv = "1.3.0"
-datafusion-common = "39.0.0"
-datafusion-expr = "39.0.0"
+datafusion = "37.1.0"
+datafusion-common = "37.1.0"
+datafusion-execution = "37.1.0"
+datafusion-expr = "37.1.0"
+datafusion-physical-expr = "37.1.0"
+datafusion-physical-plan = "37.1.0"
```

Comment:
I can force everything up to `39.0.0` if i do `default-features = false`.

However, once I do that, then there are `arrow` collisions, `pyo3` collisions, and finally a whole bunch of `lance` issues.

I'm not sure if there's a way around this, or if we're stuck on this version until lance upgrades arrow and datafusion

Reply:
I think the plan is to move benchmarks out, or maybe run them from Python, so things are pre-compiled

Reply:
@a10y we nuked lance for now. If we care about in the future we can figure out better setup. They don't upgrade dependencies as aggressively as we do.